recipes/portaudio/19.7.0/conanfile.py

```diff
@@ -74,7 +76,10 @@ def _configure_cmake(self):
             self._cmake.definitions["PA_BUILD_STATIC"] = not self.options.shared
             self._cmake.definitions["PA_BUILD_SHARED"] = self.options.shared
             self._cmake.definitions["PA_USE_JACK"] = self.options.get_safe("with_jack", False)
-            self._cmake.definitions["PA_USE_ALSA"] = self.options.get_safe("with_alsa", False)
+            if self.options.get_safe("with_alsa", False): # with_alsa=False just makes portaudio use the Linux distro's alsa
+                                                          # as a workaround to the fact that conancenter's alsa does not work,
+                                                          # at least some of the time (no devices detected by portaudio)
```

Comment:
Does this mean, that it works sometimes? If yes, under what circumstances?

Reply:
Conancenter's alsa has never worked for me, but since it was referenced by this packaged I supposed it had been tested and it had worked for somebody else so I did not want my comment to be too absolute.